Mathlib/Topology/Connected/LocallyConnected.lean

```diff
@@ -0,0 +1,150 @@
+/-
+Copyright (c) 2017 Johannes Hölzl. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Johannes Hölzl, Mario Carneiro, Yury Kudryashov
+-/
+import Mathlib.Topology.Connected.Basic
+
+/-!
+# Locally connected topological spaces
+
+A topological space is **locally connected** if each neighborhood filter admits a basis
+of connected *open* sets. Local connectivity is equivalent to each point having a basis
+of connected (not necessarily open) sets --- but in a non-trivial way, so we choose this definition
+and prove the equivalence later in `locallyConnectedSpace_iff_connected_basis`.
+-/
+-- FIXME: check copyright
```

Comment:
Could you do this? Otherwise LGTM.

Reply:
I tend to just copy the full authors list from a "parent" file to each "child" file when splitting. It's an over-approximation for both credit and blame (=asking for help when something goes wrong), but for both over-approximation seems better than under.